hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -353,7 +353,7 @@ def stop(self) -> None:
         # changes.
         insn_failed = self._err_bits and self._fsm_state == FsmState.EXEC
         if insn_failed or self.rma_req:
-            self._abort()
+            self._abort(not_status=True)
```

Comment:
should this feed in the not_status argument?

Reply:
Yes, it definitely should! I just noticed it on my machine and jumped back to the PR to see whether I was doing the right thing. Duh...